glusterd2/commands/volumes/volume-option.go

```diff
@@ -62,45 +62,66 @@ func optionSetValidate(c transaction.TxnCtx) error {
 }
 
 type txnOpType uint8
-type volumeOpType uint8
 
 const (
 	txnDo txnOpType = iota
 	txnUndo
-	volumeSet volumeOpType = iota
-	volumeReset
 )
 
 func xlatorActionDoSet(c transaction.TxnCtx) error {
-	return xlatorAction(c, txnDo, volumeSet)
+	return xlatorAction(c, txnDo, xlator.VolumeSet)
 }
 
 func xlatorActionUndoSet(c transaction.TxnCtx) error {
-	return xlatorAction(c, txnUndo, volumeSet)
+	return xlatorAction(c, txnUndo, xlator.VolumeSet)
 }
 
 func xlatorActionDoReset(c transaction.TxnCtx) error {
-	return xlatorAction(c, txnDo, volumeReset)
+	return xlatorAction(c, txnDo, xlator.VolumeReset)
 }
 
 func xlatorActionUndoReset(c transaction.TxnCtx) error {
-	return xlatorAction(c, txnUndo, volumeReset)
+	return xlatorAction(c, txnUndo, xlator.VolumeReset)
+}
+
+func xlatorActionDoStart(c transaction.TxnCtx) error {
```

Comment:
The name is misleading without any reference to volume. I suggest `xlatorActionDoVolumeStart`. Follow the same for all 4 functions you've added.

Reply:
done